projects/multiple-network-demo/src/index.ts

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
```

Comment:
There is no point to install linter if everything is disabled in files. Its better to remove it completely

Reply:
Oops yeah the linter insnt enabled for this yet .. I think I did this to remove all the warnings in my editor so I could see the typescript errors more easily :smile: i'll enable linting and fix the errors.